kclvm/ast/src/ast.rs

```diff
@@ -237,6 +250,15 @@ impl Module {
         }
         return stmts;
     }
+
+    pub fn pos_to_stmt(&self, pos: &Position) -> Option<Node<Stmt>> {
```

Comment:
Add more documents to explain why it is necessary to obtain AST from pos in reverse and its functions.